contracts/Vault1.sol

```diff
@@ -1,6 +1,23 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.4;
 
-contract Vault1 {
-    
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Wrapper.sol";
+
+contract Vault1 is ERC20Wrapper {
```

Comment:
Inheriting from ERC20Wrapper is not required, here you are creating a new token (Vault1) whereas the contract should be able to accept any ERC20 token. The best way to do that would be to take the ERC20 address in constructor and then create a token object